src/main/java/edu/wpi/grip/core/operations/OpenCVOperations.java

```diff
@@ -0,0 +1,25 @@
+package edu.wpi.grip.core.operations;
+
+
+import com.google.common.eventbus.EventBus;
+import edu.wpi.grip.core.events.OperationAddedEvent;
+import edu.wpi.grip.core.operations.opencv.MatFieldAccessor;
+import edu.wpi.grip.core.operations.opencv.NewPointOperation;
+import edu.wpi.grip.core.operations.opencv.NewSizeOperation;
+import edu.wpi.grip.generated.CVOperations;
+
+public class OpenCVOperations extends CVOperations {
```

Comment:
You are more than welcome to take a crack at doing this yourself if you'd like though. It would give you the opportunity to become familiar with the javaparser library.